ui/src/dashboards/utils/cellGetters.ts

```diff
@@ -153,12 +154,18 @@ export const getClonedDashboardCell = (
 }
 
 export const getTimeRange = (queryConfig: QueryConfig): DurationRange => {
-  return (
-    queryConfig.range || {
-      upper: null,
-      lower: TEMP_VAR_DASHBOARD_TIME,
+  if (queryConfig.range) {
```

Comment:
this is a bit confusing. is the following equivalent?

```
  const isUsingDashTime =
    queryConfig.range &&
    queryConfig.originalQuery &&
    queryConfig.originalQuery.indexOf(TEMP_VAR_DASHBOARD_TIME) !== -1

  if (isUsingDashTime || !queryConfig.range) {
    return TEMPLATE_RANGE
  }

  return queryConfig.range
```

Reply:
yeppers. that reads much better!